test/integration/metrics/metrics_test_thresholds.go

```diff
@@ -0,0 +1,15 @@
+package metrics
+
+type Thresholds struct {
+	MemAllocated       float64
+	GoRoutines         float64
+	DescribeENIApiCall float64
+}
+
+func NewThresholds() *Thresholds {
```

Comment:
Can we add a comment on how we determine the thresholds and how to update in future when we have more data and metrics?

Reply:
The comment is added. Thanks.